libsawtooth/src/client/rest.rs

```diff
@@ -218,7 +218,7 @@ impl Iterator for TransactionIter {
 
 /// A struct that represents a page of batches, used for deserializing JSON objects.
 #[derive(Debug, Deserialize)]
-pub struct Page {
```

Comment:
The commit message doesn't match the change - looks like this commit removes the `pub` modifier.